### PR TITLE
Update version of Az.HDInsight for preview release

### PR DESCRIPTION
**Pull request by cormacpayne**
## Checklist

- [x] I have read the [_Submitting Changes_](../CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by cormacpayne** [comment]
Sign job: https://azuresdkci.westus2.cloudapp.azure.com/job/psnetsign-esrp/510/

File: src/HDInsight/HDInsight/Az.HDInsight.psd1 [hunk 1]
```diff
@@ -133,7 +132,25 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Fixed miscellaneous typos across module'
+        ReleaseNotes = '* Changed Microsoft.Azure.Management.HDInsight from 2.1.0 to 5.1.0
+* Removed five cmdlets:
```
Comment (dolauli) [inline src/HDInsight/HDInsight/Az.HDInsight.psd1]: These changes look breaking ones.

Comment (dolauli) [reply src/HDInsight/HDInsight/Az.HDInsight.psd1]: @cormacpayne

**Comment by cormacpayne** [comment]
@dolauli there were breaking changes previously pushed into this branch `Az.HDInsight-preview` and this is the PR to version this module for a _preview_ release accordingly. Since these changes aren't going into the `master` branch and are going out for a preview release, we should be OK with the breaking changes 😁

File: src/HDInsight/HDInsight/Az.HDInsight.psd1 (hunk 1 above)
Comment (markcowl) [reply src/HDInsight/HDInsight/Az.HDInsight.psd1]: Yes, this is a by-design breaking change, released as a preview

**Pull request merged**